src/main/java/appeng/client/gui/implementations/GuiPatternTerm.java

```diff
@@ -28,6 +28,7 @@
 
 import appeng.api.config.ActionItems;
 import appeng.api.config.Settings;
+import appeng.api.config.ItemSubstitution;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Wrong order for 'appeng.api.config.ItemSubstitution' import. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)